util/config/gather-cray-prgenv-arguments.bash

```diff
@@ -19,6 +24,12 @@ case $1 in
     ;;
 esac
 
```

Comment:
Update the usage above? (and fail if all args aren't provided?)

Reply:
Yes, thanks for catching that.